macros/bpasync.nim

```diff
@@ -0,0 +1,147 @@
+# Copyright 2017 Xored Software, Inc.
+
+import macros, strutils
+import "../lib/macroutils"
+import uetypes, uedelegate
+
+macro blueprintAsync*(params: untyped, body: untyped): untyped =
```

Comment:
So let's change this to have similar syntax to the [{.blueprint.} pragma](https://github.com/pragmagic/nimue4/wiki/Blueprint-Libraries#blueprint-pragma).